data-accessors/projects.data-accessor.ts

```diff
@@ -0,0 +1,38 @@
+import { MongodbService } from '../services/mongodb.service'
+import { DatabaseCollections } from '../enums/database-collection.enum'
+import { Filter, Sort, WithId } from 'mongodb'
+import { Document } from 'bson'
+import { MongoDocument } from '../interfaces/mongo-document.interface'
+
+export class ProjectDataAccessor {
+    constructor(
+        private mongodbService: MongodbService
+    ) {}
+
+    public async getProjects(
+        query: Filter<WithId<Document>> = {},
+        sort: Sort = {},
+        limit = 0
+    ): Promise<MongoDocument[]> {
+        return await this.mongodbService.get(DatabaseCollections.Projects, query, sort, limit)
```

Comment:
No need to wrap the result of the promise in another promise since you aren't using the result to return some other/modified data. Just return the original promise.